Packs/Clarizen/Integrations/ClarizenIAM/ClarizenIAM_test.py

```diff
@@ -36,269 +36,268 @@ def get_outputs_from_user_profile(user_profile):
-    mocker.patch.object(client, 'update_user', return_value=USER_APP_DATA)
-
-    user_profile = IAMCommand(is_update_enabled=False).update_user(client, args)
-    outputs = get_outputs_from_user_profile(user_profile)
-
-    assert outputs.get('action') == IAMActions.UPDATE_USER
-    assert outputs.get('success') is True
-    assert outputs.get('skipped') is True
-    assert outputs.get('reason') == 'Command is disabled.'
-
-
-def test_update_user_command__allow_enable(mocker):
-    """
-    Given:
-        - An app client object
-        - A user-profile argument that contains user data
-    When:
-        - The user is disabled in the application
-        - allow-enable argument is true
-        - Calling function update_user_command
-    Then:
-        - Ensure the user is enabled at the end of the command execution.
-    """
-    client = mock_client()
-    args = {'user-profile': {'email': 'testdemisto2@paloaltonetworks.com', 'givenname': 'mock_first_name'},
-            'allow-enable': 'true'}
-
-    mocker.patch.object(client, 'get_user', return_value=DISABLED_USER_APP_DATA)
-    mocker.patch.object(client, 'get_user_by_id', return_value=DISABLED_USER_APP_DATA)
-    mocker.patch.object(client, 'update_user', return_value=USER_APP_DATA)
-    mocker.patch.object(client, 'enable_user', return_value=USER_APP_DATA)
-    mocker.patch.object(IAMUserProfile, 'map_object', return_value={})
-
-    user_profile = IAMCommand().update_user(client, args)
-    outputs = get_outputs_from_user_profile(user_profile)
-
-    assert outputs.get('action') == IAMActions.UPDATE_USER
-    assert outputs.get('success') is True
-    assert outputs.get('active') is True
-    assert outputs.get('id') == 'mock_id'
-    assert outputs.get('username') == 'mock_user_name'
-    assert outputs.get('details', {}).get('first_name') == 'mock_first_name'
-    assert outputs.get('details', {}).get('last_name') == 'mock_last_name'
-
-
-def test_disable_user_command__non_existing_user(mocker):
-    """
-    Given:
-        - An app client object
-        - A user-profile argument that contains an email of a user
-    When:
-        - create-if-not-exists parameter is unchecked
-        - The user does not exist in the application
-        - Calling function disable_user_command
-    Then:
-        - Ensure the command is considered successful and skipped
-    """
-    client = mock_client()
-    args = {'user-profile': {'email': 'testdemisto2@paloaltonetworks.com'}}
-
-    mocker.patch.object(client, 'get_user', return_value=None)
-
-    user_profile = IAMCommand().disable_user(client, args)
-    outputs = get_outputs_from_user_profile(user_profile)
-
-    assert outputs.get('action') == IAMActions.DISABLE_USER
-    assert outputs.get('success') is True
-    assert outputs.get('skipped') is True
-    assert outputs.get('reason') == IAMErrors.USER_DOES_NOT_EXIST[1]
+class TestGetUserCommand:
+    def test_get_user_command__existing_user(self, mocker):
```

Comment:
no need to repeat the name of the function you are testing when using a test class per function (duplicate information)
```suggestion
    def test_existing_user(self, mocker):
```